website/pages/blog/introducing-the-github-cloudquery-provider.md

```diff
@@ -21,63 +21,42 @@ In this tutorial, we will install CloudQuery and use it to fetch GitHub resource
 
 ## Setup
 
-First we need to [download](https://github.com/cloudquery/cloudquery/releases) the pre-compiled CloudQuery binary. We can also install it with the following [brew](https://brew.sh/) command:
-
-```bash
-brew install cloudquery/tap/cloudquery
-# After initial install you can upgrade the version via:
-brew upgrade cloudquery
-```
-
-Before running CloudQuery we need to generate a `config.yaml` via the `init` command:
-
-```bash
-cloudquery init github
-```
-
-Now that we have our `config.yaml` ready we need to add our GitHub Personal Access Token.
-The required scopes:
-
-- `read:org`
-- `read:project`
-- `public_repo`
-
-To create one, refer to [this guide](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) by GitHub.
-This token should be considered a secret and the config file should not be committed to source control.
-
-Add the token we created, and the organization(s) you want to fetch, to the configuration.
-
-```yaml
-  - name: github
-    configuration:
-      access_token: "<YOUR ACCESS TOKEN HERE>"
-      orgs: ["<YOUR ORG NAME>"]
-    resources:
-      - organizations
-      - repositories
-      - teams
-      - billing.actions
-      - billing.packages
-      - billing.storage
-      - issues
-      - hooks
-      - installations
-      - external_groups
-```
-
-## Running
-
-CloudQuery requires us to connect to a database. Either use an existing one (we change the `dsn` in the `config.yaml`), or simply create a database with the following command
-
-```bash
-docker run -p 5432:5432 -e POSTGRES_PASSWORD=pass -d postgres
-```
-
-Finally, we can execute CloudQuery to fetch all our GitHub resources. Simply run the fetch command and let the magic happen.
-
-```bash
-cloudquery fetch
-```
+- [Download CloudQuery](https://www.cloudquery.io/docs/quickstart/macOS#download-cloudquery-cli).
```

Comment:
```suggestion
- [Download CloudQuery](https://www.cloudquery.io/docs/quickstart).
```

Although we lose the ability to link to a section by doing this, I think it's better to redirect to the quickstart page appropriate for the OS you're on.